advanced/accessors/01_accessor_examples.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<blockquote>it makes it easier to perform check ...</blockquote><p>Capitalize the first <code>i</code> in sentence.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>